deps/rabbit/Makefile

```diff
@@ -119,7 +119,12 @@ define PROJECT_ENV
 	    %% interval at which connection/channel tracking executes post operations
 	    {tracking_execution_timeout, 15000},
 	    {stream_messages_soft_limit, 256},
-        {track_auth_attempt_source, false}
+        {track_auth_attempt_source, false},
+        %% Number of entries per index segment.
+        %% This value can only be changed safely
+        %% on an empty node. Default calculated
+        %% as trunc(math:pow(2,?REL_SEQ_BITS))).
+        {queue_index_segment_entry_count, 16384}
```

Comment:
This same change should also apply to `deps/rabbit/BUILD.bazel`. I would like to eliminate the double bookkeeping of the value, but I don't have a good solution just yet.

Reply:
Done, thanks! https://github.com/rabbitmq/rabbitmq-server/pull/2939/commits/dfc360c32c9a1678acd7a8d45536cdd8b7da8519